tests/test-generators.py

```diff
@@ -32,6 +32,17 @@ def parse_output(result, error_code) -> str:
     run_time = perf_counter() - start_time
     output = parse_output(result, error_code)
 
+    sys.stdout.write("running generator (hard-fork mode) ...\r")
+    start_time = perf_counter()
+    error_code3, result3 = run_gen(g, LIMIT_STACK | HARD_FORK, "generators/block-834752.hex")
+    run_time3 = perf_counter() - start_time
+    output3 = parse_output(result3, error_code3)
+    if output != output:
```

Comment:
this was a bad typo, causing the test not to catch the discrepancy in the cost of the new generator. I'll update `chia-blockchain first`